### ISSUE TITLE
Releasing the model on torch.hub?

### ISSUE THREAD

**Issue by gmberton** [YvanYin/Metric3D#76]
Could you release the models on torch.hub? It is quite simple to do and allows people to use your model with two lines of code, allowing more people to use your model and helping to spread your work!
 Once the model is on torch.hub, it is super simple to use without requiring any installation or cloning the repo, e.g. I did it for a model of my own, called CosPlace, which can be used just like this
```
import torch
model = torch.hub.load("gmberton/cosplace", "get_trained_model", backbone="ResNet50", fc_output_dim=2048)
```

**Comment by YvanYin** [comment]
Hi @gmberton , thx for your suggestion. We will add it ASAP.

**Comment by gmberton** [comment]
Hi @YvanYin do you have an estimate of when the model will be released on torch.hub?

**Comment by ZachL1** [comment]
Hi @gmberton , we have released the model on torch.hub, you can try it now.

We also provided a minimal working example in [hubconf.py](https://github.com/YvanYin/Metric3D/blob/main/hubconf.py#L122). 
Please feel free to comment if there are any problems.

**Comment by gmberton** [comment]
This is really helpful, thank you so much!
Also, I think the parameter `padding` [defined here](https://github.com/YvanYin/Metric3D/blob/main/hubconf.py#L140) is never used.
I will use it properly in a couple of weeks and let you know if I run into some issues

**Comment by ZachL1** [comment]
The parameter `padding` defined here is used as a border value when [padding](https://github.com/YvanYin/Metric3D/blob/main/hubconf.py#L146). After normalization, it is equivalent to zero padding, which is the same as the training phase.
